src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -357,7 +360,7 @@ function SuggestionMention(
             const leftString = newValue.substring(afterLastBreakLineIndex, selectionEnd);
             const words = leftString.split(CONST.REGEX.SPACE_OR_EMOJI);
             const lastWord: string = words.at(-1) ?? '';
-            const secondToLastWord = words[words.length - 3];
+            const secondToLastWord = words.at(words.length - 3);
```

Comment:
Please, make sure it doesn't break anything, cause .at() accepts negative integers while [] not. See https://github.com/Expensify/App/pull/49534#discussion_r1772650621